DEVELOPMENT.md

```diff
@@ -66,25 +81,6 @@ source ./scripts/env
 
 Make sure to source the script from the repository root.
 
-
-### Feature flagging
```

Comment:
We should only remove this if we have actually removed this functionality from our code. Otherwise I'd suggest to reword it and just make shorter, because it's still useful.